packages/dart_firebase_admin/lib/src/app/credential.dart

```diff
@@ -1,5 +1,8 @@
 part of '../app.dart';
 
+@internal
+const envSymbol = #_envSymbol;
```

Comment:
This needs to be hidden by the associated export, as we don't want this to be part of the public API